.github/workflows/graphics.yml

```diff
@@ -30,4 +30,4 @@ jobs:
           java-version: 21
       - name: Build
         working-directory: graphics-generator
-        run: ./mvnw -B install
+        run: ./mvnw -B clean verify
```

Comment:
We shouldn't need clean in a CI run, should we? If the target directory is dirty, we should fix that in source control. :) 

I did install so that later steps would have a populated maven repo. But I guess it's a fair assumption that they'd be using the built artifacts from target, rather than from the local repo.

Reply:
I just remember the meme last year going around the internet from Andres Almiray saying "friends don't let friends do mvn install" :)

> I did install so that later steps would have a populated maven repo. But I guess it's a fair assumption that they'd be using the built artifacts from target, rather than from the local repo.

Unless some later step us using this app as a dependency in its pom file, I'm not sure why populating the local maven repo matters?